plugins/woocommerce/templates/myaccount/orders.php

```diff
@@ -67,7 +67,7 @@
 
 								if ( ! empty( $actions ) ) {
 									foreach ( $actions as $key => $action ) { // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
-										echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button button ' . sanitize_html_class( $key ) . '">' . esc_html( $action['name'] ) . '</a>';
+										echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . '">' . esc_html( $action['name'] ) . '</a>';
```

Comment:
Just to confirm, since I'm not used to this codebase. We are inheriting `$wp_button_class` from a parent template? Because `$wp_button_class` is not declared in this file.

Reply:
Yup, it's defined over here: https://github.com/woocommerce/woocommerce/pull/35306/files#diff-d49488c34017d273db858b7b6eff58f22d95310991d26638e75793c6a1e98678R3213

The `echo` here is inside 2 foreach loops in the template, so didn't want to evaluate it each time, and we also want to make templates as dumb as possible, so having conditions and other code outside of them unless necessary. Which makes me think I should also remove the other conditionals 🤔